guides/common/modules/con_installing-a-server.adoc

```diff
@@ -0,0 +1,32 @@
+[id="installing-a-server_{context}"]
+= Installing a {ProjectServer}
+
+Installing an instance of {ProjectServer} on a dedicated {RHEL} 8 server is the first step to a working {Project} infrastructure.
```

Comment:
```suggestion
Installing an instance of {ProjectServer} on a dedicated server is the first step to a working {Project} infrastructure.
```

We support Debian (for Foreman) and other ELs. And 8 we do not support anymore ;-)